### DISCUSSION TITLE
Town Population screen (with the well), "Buy Monsters:" window, allow the player to prioritize the creatures he want to buy

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10294] [General]
In this window it would be nice to be able to buy in priotity either high level creatures (first level 6, then level 5, etc.) or low level creatures (first level 1, then level 2, etc.).
This is in addition to the selection of creatures to recruit proposed here: https://github.com/ihhub/fheroes2/issues/1798.

In fheroes2:

![Window change fh2 12](https://user-images.githubusercontent.com/43583994/97112669-efb6f980-16e5-11eb-9412-caf60e1512ff.png)

**Comment by ihhub** [comment]
The button is designed to buy maximum of everything. Fine-tuning it for special purposes makes no sense. If a player wants to buy specific monsters they can always buy them separately.
